spec/server_store_spec.rb

```diff
@@ -43,13 +43,13 @@
 
     describe "#update_servers" do
       it "updates the servers list" do
-        subject.update_servers(@connection)
+        subject.update_servers(@connection, "test")
```

Comment:
think it might make the spec clearer if this says "test handler" or something to that effect.